x/staging/posts/client/cli/keys.go

```diff
@@ -14,15 +16,15 @@ const (
 	flagSortBy   = "sort-by"
 	flagSorOrder = "sort-order"
 
-	FlagParentID        = "parent-id"
-	FlagAttachment      = "attachment"
-	FlagPollDetails     = "poll-details"
-	FlagPollAnswer      = "poll-answer"
-	FlagCreationTime    = "creation-time"
-	FlagDisableComments = "disable-comments"
-	FlagSubspace        = "subspace"
-	FlagHashtag         = "hashtag"
-	FlagCreator         = "creator"
+	FlagParentID      = "parent-id"
+	FlagAttachment    = "attachment"
+	FlagPollDetails   = "poll-details"
+	FlagPollAnswer    = "poll-answer"
+	FlagCreationTime  = "creation-time"
+	FlagCommentsState = "disable-comments"
```

Comment:
The flag value should be changed accordingly. I think we can use `comments`: 


```suggestion
	FlagCommentsState = "comments"
```